names.csv

```diff
@@ -1049,7 +1049,7 @@ tasha,tash,tashie
 ted,teddy
 temperance,tempy
 terence,terry
-teresa,terry,tess,tessa,tessie
+teresa,terry,tess,tessa,tessie,tere,theresa

```

Comment:
I could be wrong but I don't think "Theresa" would be a nickname or diminutive name for "Teresa".

Reply:
Oh but Theresa would be the English version of Teresa?